test/test_utilities.cpp

```diff
@@ -344,20 +344,18 @@ tf::SessionOptions GetSessionOptions() {
       ->mutable_rewrite_options()
       ->set_constant_folding(tf::RewriterConfig::OFF);
 
-  if (is_grappler_enabled()) {
-    auto* custom_config = options.config.mutable_graph_options()
-                              ->mutable_rewrite_options()
-                              ->add_custom_optimizers();
-
-    custom_config->set_name("ngraph-optimizer");
-    options.config.mutable_graph_options()
-        ->mutable_rewrite_options()
-        ->set_min_graph_nodes(-1);
-
-    options.config.mutable_graph_options()
-        ->mutable_rewrite_options()
-        ->set_meta_optimizer_iterations(tf::RewriterConfig::ONE);
-  }
+  auto* custom_config = options.config.mutable_graph_options()
```

Comment:
Won't the CPP tests now run with the optimizer, all the time?

Reply:
See below.